src/app/admin/create-vc-template/create-vc-template.component.html

```diff
@@ -7,47 +7,44 @@
       <span class="fs-16 fw-bold">{{'VERIFIABLE_CREDENTIAL_TEMPLATE' | translate}}</span>
     </div>
       
-   
-    <div class="card w-100" *ngIf = "isShow1">
-      <div class="m-5 p-5 overflow-auto">
+
+    <div class="card w-100 min-ht" *ngIf = "isShow1">
+      <div class="ml-1 p-3 w-100">
         <div class="img-hold">
-          <a href="/add-template/{{usecase}}/{{entityName}}">
-            <div class="btn img-ht">
-              <div class="thmbCol">
-                  <div class="pt-c">
-                    <a href="/add-template/{{usecase}}/{{entityName}}" class="btn submit-button pt-c">
-                       <i class="fa fa-plus text-secondary-color pt-c" aria-hidden="true"></i>
-                    </a>
-                  </div>
-                  <div class="fs-12">
-                    <b>{{'ADD_TEMPLATE' | translate}}</b>
-                  </div>
-                  <div class="fs-10 thmbCol2">
-                    <div>
-                      {{'ANY_EDIT_IN_QR_CODE' | translate}}
-                    </div>
-                    <div>
-                      {{'WILL_REFLECT_ALL_VC' | translate}}
-                    </div>
-                    <div>
-                      {{'OF_THIS_SCHEMA' | translate}}
-                     </div>
-                </div>
-              </div>
-            </div>
-          </a>
-       
-          <div class="thmbCol" *ngFor="let item of credTemp; let i=index" >           
-            <div class="wrap" >
+          <div class="thmbCol" *ngFor="let item of credTemp; let i=index" > 
+            <div class="wrap">
               <iframe id="iframe{{i}}" class="iframe-container embed-responsive-item iframeCert" ></iframe>
             </div>
             <div class="l12 fw-bold">{{item?.title}}</div>  
           </div> 
-
-         
-      </div>
+        
```

Comment:
@chaitrali-r Why you changed the Add template card position? 
cc @suvarnakale check its correct position?

Reply:
I made the change according to figma file

Reply:
As per last discussion, we are showing it at first position. discuss with @suvarnakale she worked on this changes.

Reply:
Yes changed. thanks

Reply:
yes... discussing with team we have decided its position at first